packages/analyzer/src/utils/cli.js

```diff
@@ -123,19 +124,21 @@ export const MENU = `
 @custom-elements-manifest/analyzer
 
 Available commands:
-    | Command/option   | Type       | Description                                                 | Example               |
```

Comment:
we can generate this with [command-line-usage](https://npm.im/command-line-usage). maybe in another pr